higress项目执行make build命令，编译报错 fatal: Authentication failed for 'https://github.com/mitchellh/osext/'

**环境**
测试环境系统：centos 7.9  Go环境：1.19

**问题描述**
higress项目下main分支直接执行make build编译报错：
fatal: Authentication failed for 'https://github.com/mitchellh/osext/'

**详细信息**
`
istio.io/pkg/version.buildStatus=Modified

istio.io/pkg/version.buildTag=v1.3.4-rc.1

istio.io/pkg/version.buildHub=higress-registry.cn-hangzhou.cr.aliyuncs.com/higress
Username for 'https://github.com': llahu
Password for 'https://llahu@github.com':
external/istio/pkg/config/xds/filter_types.gen.go:21:2: github.com/distribution/distribution/v3@v3.0.0-20221201083218-92d136e113cf requires
	github.com/mitchellh/osext@v0.0.0-20151018003038-5e2d6d41470f: invalid version: git ls-remote -q origin in /gopath/pkg/mod/cache/vcs/94ed57c5b21c953d93b47487113db43a5c9b69fd990329ec70dc77348c4dd443: exit status 128:
	fatal: Authentication failed for 'https://github.com/mitchellh/osext/'
external/istio/pkg/config/xds/filter_types.gen.go:22:2: github.com/distribution/distribution/v3@v3.0.0-20221201083218-92d136e113cf requires
	github.com/mitchellh/osext@v0.0.0-20151018003038-5e2d6d41470f: invalid version: git ls-remote -q origin in /gopath/pkg/mod/cache/vcs/94ed57c5b21c953d93b47487113db43a5c9b69fd990329ec70dc77348c4dd443: exit status 128:
	fatal: Authentication failed for 'https://github.com/mitchellh/osext/'
external/istio/pkg/config/xds/filter_types.gen.go:23:2: github.com/distribution/distribution/v3@v3.0.0-20221201083218-92d136e113cf requires
	github.com/mitchellh/osext@v0.0.0-20151018003038-5e2d6d41470f: invalid version: git ls-remote -q origin in /gopath/pkg/mod/cache/vcs/94ed57c5b21c953d93b47487113db43a5c9b69fd990329ec70dc77348c4dd443: exit status 128:
	fatal: Authentication failed for 'https://github.com/mitchellh/osext/'
external/istio/pkg/config/xds/filter_types.gen.go:24:2: github.com/distribution/distribution/v3@v3.0.0-20221201083218-92d136e113cf requires
	github.com/mitchellh/osext@v0.0.0-20151018003038-5e2d6d41470f: invalid version: git ls-remote -q origin in /gopath/pkg/mod/cache/vcs/94ed57c5b21c953d93b47487113db43a5c9b69fd990329ec70dc77348c4dd443: exit status 128:
	fatal: Authentication failed for 'https://github.com/mitchellh/osext/'
external/istio/pkg/config/xds/filter_types.gen.go:35:2: github.com/distribution/distribution/v3@v3.0.0-20221201083218-92d136e113cf requires
	github.com/mitchellh/osext@v0.0.0-20151018003038-5e2d6d41470f: invalid version: git ls-remote -q origin in /gopath/pkg/mod/cache/vcs/94ed57c5b21c953d93b47487113db43a5c9b69fd990329ec70dc77348c4dd443: exit status 128:
	fatal: Authentication failed for 'https://github.com/mitchellh/osext/'
`

--- a/Makefile.core.mk
+++ b/Makefile.core.mk
@@ -15,7 +15,7 @@ GO_LDFLAGS += -X $(VERSION_PACKAGE).higressVersion=$(shell cat VERSION) \
 
 GO ?= go
 
-export GOPROXY ?= https://proxy.golang.com.cn,direct
+export GOPROXY ?= https://proxy.golang.org,direct
 
 TARGET_ARCH ?= amd64
 

